Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -190,6 +192,19 @@ readNoPreambleDicom(std::ifstream & file) // NOTE: This file is duplicated in it
   return true;
 }
 
+static void convert_YCbCr(unsigned char * p, const size_t len, const uint8_t t)
+{
+  if (len % 3 != 0) return;
```

Comment:
This should probably be an exception, not a silent `return`.

Reply:
Done

Reply:
I've done an exception, please look, may be replace with warning, probably invalid data would be caught somewhere else. Otherwise can not harm too much.

Reply:
Exactly, it should not come to this point with `len % 3 != 0`.